tiktok.cs

```diff
@@ -5,394 +5,105 @@ namespace First_project
-                {
-                    Console.WriteLine("YouWin!!");
-                    return true;
-
-                }
-            }
-            if (board[xGuess + 1, yGuess - 1] == 2)
-            {
-                if (board[xGuess + 2, yGuess - 2] == 2)
-                {
-                    Console.WriteLine("YouWin!!");
-                    return true;
-                }
-                if (board[xGuess - 1, yGuess + 1] == 2)
-                {
-                    Console.WriteLine("YouWin!!");
-                    return true;
-
-                }
-            }
-            if (board[xGuess, yGuess - 1] == 2)
-            {
-                if (board[xGuess, yGuess - 2] == 2)
-                {
-                    Console.WriteLine("YouWin!!");
-                    return true;
-                }
-                if (board[xGuess, yGuess + 1] == 2)
-                {
-                    Console.WriteLine("YouWin!!");
-                    return true;
-
-                }
-            }
-            if (board[xGuess - 1, yGuess - 1] == 2)
-            {
-                if (board[xGuess - 2, yGuess - 2] == 2)
-                {
-                    Console.WriteLine("YouWin!!");
-                    return true;
-                }
-                if (board[xGuess + 1, yGuess + 1] == 2)
-                {
-                    Console.WriteLine("YouWin!!");
-                    return true;
-
-                }
-            }
-            if (board[xGuess - 1, yGuess] == 2)
-            {
-                if (board[xGuess - 2, yGuess] == 2)
-                {
-                    Console.WriteLine("YouWin!!");
-                    return true;
-                }
-                if (board[xGuess + 1, yGuess] == 2)
-                {
-                    Console.WriteLine("YouWin!!");
-                    return true;
-
-                }
-            }
-            if (board[xGuess - 1, yGuess + 1] == 2)
-            {
-                if (board[xGuess - 2, yGuess + 2] == 2)
-                {
-                    Console.WriteLine("YouWin!!");
-                    return true;
-                }
-                if (board[xGuess + 1, yGuess - 1] == 2)
-                {
-                    Console.WriteLine("YouWin!!");
-                    return true;
-
-                }
-            }
-            if (board[xGuess, yGuess + 1] == 2)
-            {
-                if (board[xGuess, yGuess + 2] == 2)
-                {
-                    Console.WriteLine("YouWin!!");
-                    return true;
-                }
-                if (board[xGuess, yGuess - 1] == 2)
-                {
-                    Console.WriteLine("YouWin!!");
-                    return true;
-
-                }
-            }
-            return false;
-        }
-
-            static void Main(string[] args)
-        {
-
-            
             bool play = true;
-            string playString;
-            while (play == true)

```

Comment:
play is a boolean so theres no need to say while(play == true), just while(play). A while loop takes a boolean as argument.